vnext/ReactUWP/Views/ViewControl.h

```diff
@@ -3,24 +3,37 @@
 
 #pragma once
 
+#include <Views/ViewPanel.h>
 #include <Views/ShadowNodeBase.h>
 
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.UI.Xaml.h>
+#include <winrt/Windows.UI.Xaml.Automation.h>
 #include <winrt/Windows.UI.Xaml.Automation.Peers.h>
+#include <winrt/Windows.UI.Xaml.Automation.Provider.h>
 #include <winrt/Windows.UI.Xaml.Controls.h>
 #include <winrt/Windows.UI.Xaml.Media.h>
 
+namespace winrt {
+  using namespace Windows::Foundation;
+  using namespace Windows::UI::Xaml;
+  using namespace Windows::UI::Xaml::Automation;
+  using namespace Windows::UI::Xaml::Automation::Peers;
+  using namespace Windows::UI::Xaml::Automation::Provider;
+  using namespace Windows::UI::Xaml::Controls;
+  using namespace Windows::UI::Xaml::Media;
+}
+
 namespace react {
```

Comment:
Similar to a discussion in another review, we keep "using namespace" declarations out of header files so they don't pollute all the files that include them. It's definitely awkward here with the winrt pattern, but we've been fairly consistent about it.

Reply:
I'll switch it back for now, but at some point this entire project needs an overhaul in how it consumes and produces cppwinrt. Awkward and inconsistent is the least of it.